tests/logictest/README.md

```diff
@@ -59,6 +59,37 @@ docker build -t sqllogic/test:latest .
 - ADDITIONAL_HEADERS (for security scenario)
 3. docker run --name logictest --rm --network host public.ecr.aws/k3y0u5f2/sqllogic/test:latest
 
+## Write logic test tips
+
+1. skipif  help you skip test of given handler
+```
+skipif clickhouse
```

Comment:
what` s the diff of skipif and labels

Reply:
> what` s the diff between skipif and labels

Labels use to get the query results, no more meaning now. skipif and onlyif follows the name of handler(mysql, http, clickhouse for now)